docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  username text, -- the user name of the user, known as the `login` field in GitHub API responses
+  user_type text check ( user_type IN ('User', 'Organization') )
+);
+
+CREATE TABLE IF NOT EXISTS github_repositories (
+  id serial PRIMARY KEY,
+  repo_name text,
+  owner integer references github_users ON DELETE CASCADE
+);
+
+-- Although pull requests have different features from
+-- regular issues, they can be treated the same for our purposes
+CREATE TABLE IF NOT EXISTS github_issues (
+  id serial PRIMARY KEY,
+  issue_number integer,
+  repo integer references github_repositories ON DELETE CASCADE,
+  issue_state text check ( issue_state IN ('open', 'closed') )
+);
+
+/* 
+  TODO:
+    * Add policies to restrict the users who can write into these
```

Comment:
We can supply our services a secret `service role key`, authorize using Postgre function (not sure how to implement yet)

Reply:
I can file a future issue to address this, like how @humphd suggested.